index.html

```diff
@@ -115,6 +115,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Dex</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/DexTheDev_" target="_blank">@DexTheDev_</a>
+          </p>
+          <p class="about">I love sushi and video games. My cat is black. Coding makes me feel like a genius.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://fonts.google.com/" target="_blank" title="Google Fonts">
+                 Google Fonts
+                </a>
+              </li>
+              <li>
+                <a href="https://coolors.co/" target="_blank" title="A palette generator">
+                  Coolors - A Palette Generator
+                </a>
+              </li>
+              <li>
+                <a href="https://roadmap.sh/" target="_blank" title="Raodmap">
+                  Roadmap - A guide for web development careers
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change this 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` comment to
`<!-- ________ YourName Contributor card START ________ -->` as well.